<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>320</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="LabelWidget" name="showWidget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>420</width>
      <height>300</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="addButton">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>20</y>
      <width>31</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="delButton">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>80</y>
      <width>31</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="wideButton">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>190</y>
      <width>31</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;-&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="narrowButton">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>250</y>
      <width>31</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>&gt;-&lt;</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>LabelWidget</class>
   <extends>QWidget</extends>
   <header>LabelWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
